config/base/kong-ingress-dbless.yaml

```diff
@@ -74,6 +74,12 @@ spec:
         - name: KONG_ROUTER_FLAVOR
           value: expressions
         lifecycle:
+          postStart:
+            exec:
+              command:
+              - bash  
+              - -c
+              - "nofilelimit=$(prlimit --nofile --noheadings -oSOFT); if [ $nofilelimit -lt 4096 ]; then prlimit --nofile=4096 --pid=1; fi"
```

Comment:
I'm surprised this works given that by default `kong/kong-gateway` runs the container as `kong` user, and only `root` should be able to change the resource limits of a process? 🤔 

```
lima@lima-lima:/$ prlimit --nofile=4096 --pid=1
prlimit: failed to set the NOFILE resource limit: Operation not permitted
```

```
root@lima-lima:/# prlimit --nofile=4096 --pid=1
root@lima-lima:/#
```

If it works it works, let's leave it as is.